commandapi-documentation-code/src/main/java/dev/jorel/commandapi/examples/java/Examples.java

```diff
@@ -1330,6 +1330,30 @@ void resultingcommandexecutor3(){
 /* ANCHOR_END: argumentsayhicmd2 */
 }
 
+{
+/* ANCHOR: argumentrate */
+new CommandAPICommand("rate")
+    .withOptionalArguments(new StringArgument("topic").combineWith(new IntegerArgument("rating", 0, 10)))
+    .withOptionalArguments(new PlayerArgument("target"))
+    .executes((sender, args) -> {
+        String topic = (String) args.get("topic");
+        if(topic == null) {
+            sender.sendMessage("You didn't give a rating");
```

Comment:
This default case could be replaced with a help message like Skepter suggested over here: https://github.com/JorelAli/CommandAPI/pull/394#issuecomment-1372559436. Maybe something like:
```java
sender.sendMessage(
	"Usage: /rate <topic> <rating> <player>(optional)",
	"Select a topic to rate, then give a rating between 0 and 10",
	"You can optionally add a player at the end to give the rating to"
)
```
And of course, make sure the same message is used in the Kotlin and DSL examples.